RobotCode/snobot2018/src/org/snobot/Properties2018.java

```diff
@@ -25,14 +25,14 @@
     public static final DoubleProperty sAUTON_DEFAULT_SPEED = new DoubleProperty("AutonDefaultSpeed", .5);
     public static final DoubleProperty sGO_TO_XY_KPD = new DoubleProperty("GoToXYKPD", 0.015);
     public static final DoubleProperty sGO_TO_XY_KPA = new DoubleProperty("GoToXYKPA", .05);
-    public static final IntegerProperty sDEFAULT_SWITCH_POSITION = new IntegerProperty("DefaultSwitchPosition", 0);
     public static final DoubleProperty sMIN_VOLTAGE = new DoubleProperty("MinVoltage", 0);
     public static final DoubleProperty sMAX_VOLTAGE = new DoubleProperty("MaxVoltage", 5);
-    public static final StringProperty sPOSITION_ONE_FILE = new StringProperty("PositionOneFile", "SwitchPositionOne");
-    public static final StringProperty sPOSITION_TWO_FILE = new StringProperty("PositionTwoFile", "SwitchPositionTwo");
-    public static final StringProperty sPOSITION_THREE_FILE = new StringProperty("PositionThreeFile", "SwitchPositionThree");
-    public static final StringProperty sPOSITION_FOUR_FILE = new StringProperty("PositionFourFile", "SwitchPositionFour");
-    public static final StringProperty sPOSITION_FIVE_FILE = new StringProperty("PositionFiveFile", "SwitchPositionFive");
+    public static final IntegerProperty sDEFAULT_SWITCH_POSITION = new IntegerProperty("DefaultSwitchPosition", 0); // Use chooser
+    public static final StringProperty sAUTON_MODE_1_FILE = new StringProperty("AutonMode1", "RealAutonomous/ScoreOnSwitchTrajectory/ScoreOnSwitch_LeftToLeftSwitch");
```

Comment:
It's ok to have specific commands for each switch positon but I thought that the PositionOneFile, etc., was a good approach because you could change the contents of the files to change the commands.  This change is more straight forward but not as flexible.